src/pages/support/support.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { HTMLAttributes, ReactNode } from 'react';
+import classNames from 'classnames';
+
+import { centerColumn } from '@bit/bit.base-ui.layout.page-frame';
+
+import { Hero } from '../../sections/support-page/hero';
+
+import styles from './support.module.scss';
+import { SupportDevelopers } from '../../sections/support-page/support-developers';
+import { SupportChannels } from '../../sections/support-page/support-channels';
+import { GlobalEnterprise } from '../../sections/support-page/global-enterprise';
+import { SupportCta } from '../../sections/support-page/support-cta';
+import { TransparentPackages } from '../../sections/support-page/transparent-packages';
+
+export type SupportPageProps = {
+	table?: ReactNode;
+} & HTMLAttributes<HTMLDivElement>;
+
+/**
+ * A full, responsive page, detailing Bit's offering for support.
+ * @name SupportPage
+ */
+export function SupportPage(props: SupportPageProps) {
+	return (
+		<div className={styles.supportPageStyles}>
```

Comment:
please pass {...rest} to this div

Reply:
and please move section layouts to from the individual sections to this components.
(Specifically, the spacing between sections, and `centerColumn`)

This will help you make fine tuning a lot easier, and will probably make the spacing more consistent. You can use the layout/spacing component